app/components/pokemon-item/pokemon-item.jsx

```diff
@@ -23,8 +31,8 @@ export const PokemonItem = ({ pokemon }) => {
         <p>{pokemon?.name}</p>
       </div>
       <div className={styles['button-container']}>
-        <button onClick={() => setIsFrontImage(true)}>Front</button>
-        <button onClick={() => setIsFrontImage(false)}>Back</button>
+        <button onClick={() => setFront()}>Front</button>
+        <button onClick={() => setBack()}>Back</button>
```

Comment:
You don't have to pass them as inline functions. `onClick={setFront}`. Similar for the other one.